core/unit_test/TestWithoutInitializing.hpp

```diff
@@ -215,3 +215,21 @@ TEST(TEST_CATEGORY, view_alloc_exec_space_int) {
   ASSERT_TRUE(success);
   listen_tool_events(Config::DisableAll());
 }
+
+TEST(TEST_CATEGORY, deep_copy_zero_memset) {
+  using namespace Kokkos::Test::Tools;
+  listen_tool_events(Config::DisableAll(), Config::EnableKernels(),
+                     Config::EnableAllocs());
```

Comment:
I forgot to remove it when copying from somewhere else but it doesn't really matter since we are not checking for it anyway.
Do you want me to change?

Reply:
Yes please fix and cleanup history

Reply:
Done.